lib/src/model.dart

```diff
@@ -5273,14 +5319,14 @@ class PackageGraph {
     List<String> messageParts = [warningMessage];
     if (warnable != null) {
       messageParts
-          .add("${warnablePrefix} ${warnableName}: ${warnable.location ?? ''}");
+          .add("${warnablePrefix} ${warnableName}: ${warnable.location}");
```

Comment:
Again: remove braces?  Up to you obviously but this is at least flutter style...

https://dart-lang.github.io/linter/lints/unnecessary_brace_in_string_interps.html

Reply:
done